src/content/insertable-streams/webgpu/js/multi_video_worker.js

```diff
@@ -1,18 +1,24 @@
-importScripts('./multi_video_main.js')
+// eslint-disable-next-line no-undef
+importScripts('./multi_video_main.js');
```

Comment:
You should probably add importScripts to https://github.com/webrtc/samples/blob/gh-pages/.eslintrc.js

It might be worthwhile to add the WebGPU globals there as well -- looks like it's just GPUBufferUsage that's needed here.